mmv1/third_party/terraform/acctest/bootstrap_test_utils.go.tmpl

```diff
@@ -2148,29 +2150,34 @@ func bootstrapSharedTestTagKey(t *testing.T, testId, parent string, obj map[stri
 		t.Fatalf("Error getting shared tag key %q: %s", sharedTagKey, err)
 	}
 
-	return sharedTagKey
+	return map[string]interface{}{
```

Comment:
given these are both strings, would it make sense to instead use:

```suggestion
	return map[string]string{
```

or maybe even a struct?

Reply:
Good point, updated to use `map[string]string`